en/docs/learn/consume-api/discover-apis/search.md

```diff
@@ -74,6 +74,11 @@ You can search for APIs in the API Publisher or Developer Portal in the followin
 <p>A property can be defined to an API at the time it is created or later. There can be APIs without properties as this parameter is optional. <strong><br />
 </strong></p></td>
 </tr>
+<tr class="even">
+<td>By API category</td>
+<td><p><strong>api-category:xxxx</strong> . For example, api-category:test</p>
+<p>An API category can be used to group APIs based on any criteria. API categories can be defined from the Admin Portal and can be assigned to an API from the Design Configurations section of the Publisher Portal.</p></td>
```

Comment:
Isn't it better to add 'Refer API Categories'[https://apim.docs.wso2.com/en/latest/develop/customizations/customizing-the-developer-portal/customize-api-listing/api-category-based-grouping/#api-category-based-grouping] for more information' and remove the 'API categories can be defined from the Admin Portal and can be assigned to an API from the Design Configurations section of the Publisher Portal'